app/Actions/Docker/GetContainersStatus.php

```
@@ -199,10 +199,33 @@ public function handle(Server $server, ?Collection $containers = null, ?Collecti
-                                $database->update(['last_online_at' => now()]);
+                                $updateData = ['last_online_at' => now()];
+
+                                // Update restart tracking even if status unchanged
+                                if ($restartCount > $previousRestartCount) {
+                                    $updateData['restart_count'] = $restartCount;
+                                    $updateData['last_restart_at'] = now();
+                                    $updateData['last_restart_type'] = 'crash';
+                                }
+
+                                $database->update($updateData);
                             }
```

PHPMD (2.15.0)

The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)